book/lessons/multicolor-led-1/lesson.md

```diff
@@ -1,5 +1,125 @@
 # Multi-color LED
 
-This lesson is not yet developed. 😿
+## Getting started
+For this exercise, you will need the following:
+* Raspberry Pi 400 computer
+* Sparkfun Qwiic pHAT Extension
+* One multi-color LED
+* Four M-to-F Jumper Wire (1 Black, 1 Red, 1 Green, 1 Blue)
+* Four 330 Ohm resistor
 
-![Completed breadboard](images/breadboard-1.jpg)
\ No newline at end of file
+## Connecting the LED
+
+1. With the pHAT already connected (See Blinking LED Lesson above):
+    * View the multi-colored LED, the longest pin is the anode.
+[LED LESSON ON LED CATHODE/ANODE](https://www.admfactory.com/rgb-led-on-raspberry-pi-using-python/)
+    * The other pins represent RGB, respectively, so we read the pin as follows:
+Red, Anode, Green, Blue.
+We will have the jumper wires complement each pin color for easier coordination, we will also line up these pins on the GPIO pins.
+
+2. Install RGB-LED.
+    * Insert anode vertically into the Column I, Row 21.
+
+    ![connection-to-breadboard](images/to-breadboard.jpg)
+
+3. Gather your resistors to allow a steady current for our LED.
+    * Insert the resistor with the orange ends facing toward the LED pins, the first will will go into the Column H, Row 20.
```

Comment:
It shouldn't matter which end of the resistors are plugged in where as they don't have directionality. Was there a specific reason you wanted to include that here @Oddant1?

Reply:
I just copy pasted that from the write-up Franklin gave me and then once overed it to make sure nothing seemed completely incorrect. I don't think the direction matters.

Reply:
Ok, thanks! I updated the text to remove the comments about directionality.